src/components/Book.js

```diff
@@ -2,27 +2,61 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { deleteBookFromApi } from '../redux/books/booksSlice';
+import ProgressBar from './ProgressShow';
 
 const Book = ({ book }) => {
   const dispatch = useDispatch();
   return (
-    <>
-      <p key={book.id}>
-        <strong>{book.title}</strong>
-        {' '}
-        <span>by</span>
-        {' '}
-        <strong>{book.author}</strong>
-      </p>
-      <button
-        type="button"
-        onClick={() => {
-          dispatch(deleteBookFromApi(book.id));
-        }}
-      >
-        Delete
-      </button>
-    </>
+    <div className="book-Card">
+      <div key={book.id} className="book-info">
+        <p className="class-">{book.category}</p>
+        <p className="Title">{book.title}</p>
+        <p className="Author">{book.author}</p>
+        <ul>
+          <li>
+            <button className="Comments" type="button">
+              Comments
+            </button>
+          </li>
+          <div className="level-one" />
+          <li>
+            <button
+              className="Remove"
+              type="button"
+              onClick={() => {
+                dispatch(deleteBookFromApi(book.id));
+              }}
+            >
+              Remove
+            </button>
+          </li>
+          <div className="level-one" />
+          <li>
+            <button className="Edit" type="button">
+              Edit
+            </button>
+          </li>
+        </ul>
+      </div>
+      <div className="item-progress">
+        <ProgressBar />
+        <div className="task-progress">
+          <p className="Complete-Task">
+            {Math.floor(Math.random() * (99 - 0)) + 0}
+            <span>%</span>
+          </p>
+          <p className="Completed">Completed</p>
+        </div>
```

Comment:
- [ ] Kindly make sure you use the correct color for the grayed-out part of the circle. 🙂️
- [ ] Please use the correct color and `font-family` for the text `Completed`. Could you remove the border on it too? 🧐️

| Your design                                                             | Zeplin design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/230082329-bd9e1a55-f2f8-4571-ad08-3f1acecd33c8.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/230082325-0a177fc8-8edc-4b70-a64d-d6d7f18cfc3f.png" width="300">|

Reply:
- [ ] As pointed out earlier, please kindly implement the `gray` part of the circle and add the correct `font-family` to the text (`Montserrat`). 👍 This will really show that we pay attention to details. Thanks. 🙏 

## Your design
<img width="195" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/230341439-35cdf8f6-ad59-4da1-8505-fa5823516f0b.png">